
# GITHUB ISSUES REPORT FOR microsoft/vscode-python


Generated on 2022-08-02 using: stale=30, all=0


Only showing items that are new to report in past 1 day(s)


## FOR ISSUES THAT ARE MARKED AS BUGS:


### Issues in vscode-python that have comments from OP after last team response:


\* [19486](https://github.com/microsoft/vscode-python/issues/19486 " Test Discovery fails with ModuleNotFoundError while command is working on console"): OP responded 0 days ago but team last responded 14 days ago

\* [19595](https://github.com/microsoft/vscode-python/issues/19595 "python unittests in one file report wont run"): OP responded 0 days ago but team last responded 0 days ago

---

## FOR ISSUES THAT ARE NOT MARKED AS BUGS:


### Issues in vscode-python that need a response from team:


\* [19597](https://github.com/microsoft/vscode-python/issues/19597 "Python Debugger Doesn't Stop on Breakpoint"): needs an initial team response (0 days old)

\* [19598](https://github.com/microsoft/vscode-python/issues/19598 "data viewer is broken, fails to load data while debugging"): needs an initial team response (0 days old)

\* [19601](https://github.com/microsoft/vscode-python/issues/19601 "VS Code preview does not reload code on restart"): needs an initial team response (0 days old)

\* [19604](https://github.com/microsoft/vscode-python/issues/19604 "data viewer right click is broken"): needs an initial team response (0 days old)

\* [19605](https://github.com/microsoft/vscode-python/issues/19605 "One error pops up &quot;The Pylance server crashed 5 times in the last 3 minutes. The server will not be restarted. &quot;"): needs an initial team response (0 days old)

### Issues in vscode-python that have comments from 3rd party after last team response:


\* [1248](https://github.com/microsoft/vscode-python/issues/1248 "Can't expand to check the data of a Variable when debugging pytorch code"): 3rd party responded 0 days ago but team last responded 737 days ago

\* [18073](https://github.com/microsoft/vscode-python/issues/18073 "Drop the soft dependency on Jupyter"): 3rd party responded 0 days ago but team last responded 188 days ago
